### ISSUE TITLE
How to load gitfast plugin?

### ISSUE THREAD

**Issue by ztlevi** [zdharma/zinit#477]
Hi, I came across this amazing plugin!

But I got some trouble when loading git-prompt plugin. I tried cp but not working.
```
zplugin ice cp"git-prompt.sh -> git-prompt.sh"
zplugin snippet OMZP::gitfast
```

Error:
```
/Users/ztlevi/.zinit/snippets/OMZP::gitfast/OMZP::gitfast:source:1: no such file or directory: /Users/ztlevi/.zinit/snippets/OMZP::gitfast/git-prompt.sh
```

**Comment by Mellbourn** [comment]
I have this problem too. To me, it seems to be part of a general problem: `zplugin snippet OMZP::nnn` only downloads one file, the one named `nnn.plugin.zsh`, but many Oh-My-Zsh plugins contain several files, and won't work without all of them being downloaded.

In your case, [`gitfast`](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/gitfast) plugin contains the [`git-prompt.sh`](https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/gitfast/git-prompt.sh) file, but that is not downloaded by `zinit snippet OMZP::gitfast`.

I have seen the problem several times. For example, the `osx` plugin has the same issue. This will fail:

```
zinit ice wait'1' lucid
zinit snippet OMZP::osx
```

with this error

```
/Users/klas.mellbourn/.zinit/snippets/OMZP::osx/OMZP::osx:source:236: no such file or directory: /Users/klas.mellbourn/.zinit/snippets/OMZP::osx/music
/Users/klas.mellbourn/.zinit/snippets/OMZP::osx/OMZP::osx:source:239: no such file or directory: /Users/klas.mellbourn/.zinit/snippets/OMZP::osx/spotify
````

The documentation mentions that I could use `svn` to download an entire directory. Is that what I'm supposed to do? It requires me to install `svn` which feels like overkill.

**Comment by iloveitaly** [comment]
[Here's the snippet](https://github.com/iloveitaly/dotfiles/blob/c30fcc43ee513a9a7ec2653d24506659a71ead82/.zsh_plugins#L6) that worked for me:

```
zinit ice svn wait"0" lucid; zinit snippet OMZ::plugins/gitfast
```

**Comment by Mellbourn** [comment]
Yeah, if I use that snippet, it requires me to install Subversion. I'd really prefer not to. (Seems a bit ironic that you are required to install subversion to get a git tool?)

**Comment by tylerw** [comment]
For anyone wanting to use OMZ's osx, I load it the following way:
```
  zinit wait lucid for \
    atclone"touch music spotify" atpull"%atclone" \
    atload"alias open_command=open"\
      'OMZP::osx/osx.plugin.zsh'
```

(I don't use `music` or `spotify`. And the `open_command` alias is necessary for the `ofd` alias.)
